components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -111,13 +127,11 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
 
   closeEvent(): void {
     this.modifyID = false;
-    this.checkedProjectIDs = [];
     this.close.emit();
   }
 
   createObject(): void {
     this.createClicked.emit();
-    this.checkedProjectIDs = [];
   }
```

Comment:
Moved the resetting to the opener instead of the closers (which is where all the other resetting is).